package.json

```diff
@@ -132,6 +132,7 @@
     "**/sha.js": "^2.4.12",
     "**/@types/node": "~20.10.5",
     "**/ansi-regex": "^5.0.1",
+    "**/validator": "^13.15.23",
```

Comment:
i am pretty sure we do not need a package resolution for this:

Try doing this:

- remove this package resolution.
- on `yarn.lock`, remove lines 27521-27524
- run `yarn osd bootstrap`.
- verify on yarn.lock what the version it resolves to